app/views/calculation/whatNext/whatNextNonSaGain.scala.html

```diff
@@ -31,7 +31,7 @@ <h1 class="govuk-heading-xl">@Messages("calc.resident.whatNextNonSa.gain.title")
     <p id="details-two" class="govuk-body">@Messages("calc.resident.whatNextNonSa.gain.detailsTwo")</p>
 
     <p id="report-now">
-        <a class="govuk-button" href=@iFormUrl>@Messages("calc.resident.whatNextNonSa.gain.reportNow")</a>
+        <a class="govuk-button" data-module="govuk-button" role="button" href=@iFormUrl>@Messages("calc.resident.whatNextNonSa.gain.reportNow")</a>
```

Comment:
It is better to use GovukButton than maintain these attributes within raw HTML
```suggestion
        @govukButton(Button(
            href = iFormUrl, 
            content = Text(messages("calc.resident.whatNextNonSa.gain.reportNow"))
        ))
```